### ISSUE TITLE
@syncfusion/ej2-vue-schedule Module eventSettings.template not working for safari

### ISSUE THREAD

**Issue by motea927** [syncfusion/ej2-vue-ui-components#58]
In @syncfusion/ej2-vue-schedule Module

Hi, I have a code like this:

App.vue:

```
import calendarViewEventTemplate from '../../components/calendarView/calendarViewEventTemplate.vue'

eventSettings: {
  dataSource: [],
  template () {
    return {
      template: calendarViewEventTemplate
    }
}
```
----------------------------------------------------
When using chrome, It works perfect,

When I use safari this is not working,

How can I resolve it?

**Comment by motea927** [comment]
This issue only occur when using e-resources,

When using chrome, It works perfect,

When I use safari this is not working,


the complete code: https://github.com/motea927/ej2-vue-schedule-issue

**Comment by motea927** [comment]
After tested, this error occurs in  1. iPhone 11's chrome 2. iPhone 11's safari 3. Mac air m1's safari,

And working in 1. Android's chrome 2. Mac air m1's chrome

**Comment by vinithadevi** [comment]
Dear Customer, 

Greetings from Syncfusion Support..! 

We have validated your shared “When I use safari this is not working” query at our end and for that, we have ensured the eventTemplate online sample in the Browser stack link and the templates are properly rendering in the Safari Browser 14 version.  

Online Sample: https://ej2.syncfusion.com/vue/demos/#/material/schedule/events-template.html 
UG link: https://ej2.syncfusion.com/vue/documentation/schedule/appointments/#using-template 

Kindly find the below details of the Browser stack link, 

Safari version: 14 version 
Device name: macOS Big Sur 
BrowserStack link: https://live.browserstack.com/ 

Kindly try the above solution and if the issue still persists at your end kindly share the below details to serve you better? 

	iOS device name and version.
	Safari browser version.

We will happy to assist you. 
  
Regards, 
Vinitha
